crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -445,6 +451,28 @@ def delete_pacemaker_resources(node_name)
         end
       end
 
+      # Evacuate all routers away from the specified network node to other
+      # available network nodes. The evacuation procedure it started on the
+      # specified controller node
+      def evacuate_network_node(controller, network_node, delete_namespaces = false)
```

Comment:
The controller node is actually not the target. Well, it might be some cases (the standard single-two-node-cluster setup). But in most other cases it isn't.